src/reference/asciidoc/aggregator.adoc

```diff
@@ -114,6 +114,19 @@ This method will be invoked for aggregating messages as follows:
 
 NOTE: In the interest of code simplicity, and promoting best practices such as low coupling, testability, etc., the preferred way of implementing the aggregation logic is through a POJO, and using the XML or annotation support for configuring it in the application.
 
+If the `MessageGroupProcessor` 's `process` method returns a collection, it must be a collection of `Messge<?>` s.
```

Comment:
> it must be a collection of `Messge<?>` 

??
Please, elaborate more. It isn't a law: we really can return a `Collection<Object>` of payload, like it is done with `DefaultAggregatingMessageGroupProcessor`.
Maybe like this:

> If the `MessageGroupProcessor` 's `process` method returns a `Collection<Messge<?>>`, the messages are released individually.

?